### DISCUSSION TITLE
book binder

### DISCUSSION THREAD

**Discussion by satarics** [cataclysmbn/Cataclysm-BN#2292] [General]
[Google translate]
not long ago I flipped through "Hitchhiker's Guide to the Cataclysm"
and found one thing with the name "book binder" decided to see if it was in CataBN and it turned out that it was not.
My suggestion/wish is to port this item from DDA to BN.
https://nornagon.github.io/cdda-guide/#/item/book_binder

**Comment by olanti-p** [comment]
It's being worked on in https://github.com/cataclysmbnteam/Cataclysm-BN/pull/2170
